templates/reports/500.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'layout/master.html' %}
+
+{% block content %}
+
+<div id="messages">
+<p>Page not found.</p>
+</div>
+
+<p>Sorry, the page you are attempting to visit does not exist.</p>
+
+<p>It may have been removed.</p>
+
+{% endblock %}
```

Comment:
Why is this template called 500.html? The content looks more appropriate for a 404 response.